lib/jsdom/living/events/HashChangeEvent.webidl

```diff
@@ -1,6 +1,7 @@
-[Exposed=Window,
- Constructor(DOMString type, optional HashChangeEventInit eventInitDict)]
+// https://html.spec.whatwg.org/multipage/browsing-the-web.html#the-hashchangeevent-interface
+[Exposed=Window]
 interface HashChangeEvent : Event {
+  constructor(DOMString type, optional HashChangeEventInit eventInitDict = {});
```

Comment:
Fun&nbsp;fact: [this&nbsp;is&nbsp;still&nbsp;wrong in&nbsp;the&nbsp;HTML&nbsp;spec](https://html.spec.whatwg.org/multipage/browsing-the-web.html#the-hashchangeevent-interface), so&nbsp;I’m&nbsp;fixing&nbsp;it&nbsp;in&nbsp;<https://github.com/whatwg/html/pull/5189>.